dockeradapter/filterdockerstream.go

```diff
@@ -24,8 +25,14 @@ const (
 	stdWriterSizeIndex = 4
 )
 
+// frameHeader holds the steam type and payload size for a Docker stream frame
```

Comment:
stream frame is correct here I think, the stream type is part of the prefix for a frame